app/controllers/issue_tta_reports_controller.rb

```diff
@@ -42,6 +42,6 @@ def tta_report_display_id
   end
 
   def issue_id
-    params[:issue_id]
+    params[:issue_id_tta]
```

Comment:
[question] why did this need to change?

Reply:
When I started working on this, there was a hidden input with id="issue_id" for the TTA form. I copied the whole section and started working on monitoring and totally didn't think about changing that ID.  Then I got this:

https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/pull/214#issuecomment-954893194

No duplicate ids -- so I figured I would rather change them both to be `issue_id_{system}` rather than leave TTA as is.  I'm open to other ideas, though, if you would prefer something else!

Reply:
Ah, that makes sense. I'm good with doing things this way